data_sources/envo_ontology.yaml

```diff
@@ -0,0 +1,5 @@
+name: envo_ontology
+category: ontology
+namespace_title: Environment Ontology
+home_url: http://www.obofoundry.org/ontology/envo.html
+data_url: http://www.obofoundry.org/ontology/envo.html
```

Comment:
I actually got the data by trolling GitHub releases and dropping anything that wasn't valid yaml (which was most of the files).

Reply:
See https://github.com/kbaseIncubator/relation_engine_incubator/tree/master/relation_engine/ontologies/obograph/helper_scripts

Reply:
Okay I changed the data source url for Envo to be their github releases page
Their release descriptions are pretty wild